src/NLog/Internal/PropertyHelper.cs

```diff
@@ -393,6 +391,27 @@ private static bool TryCreateCollectionObject(Type collectionType, string valueR
             return false;
         }
 
+        /// <summary>
+        /// Attempt to reuse the HashSet.Comparer from the original HashSet-object (Ex. StringComparer.OrdinalIgnoreCase)
+        /// </summary>
+        private static object[] CreateHashSetConstructorArgs(object obj, PropertyInfo propInfo)
```

Comment:
Proposal, rename to FindComparer as the array is never longer than 1

(And adjust body ;)

Reply:
Fixed